pkg/gcc/send_side_bwe.go

```diff
@@ -0,0 +1,201 @@
+package gcc
+
+import (
+	"sync"
+	"time"
+
+	"github.com/pion/interceptor"
+	"github.com/pion/rtcp"
+	"github.com/pion/rtp"
+)
+
+const transportCCURI = "http://www.ietf.org/id/draft-holmer-rmcat-transport-wide-cc-extensions-01"
+
+// Pacer is the interface implemented by packet pacers
+type Pacer interface {
+	interceptor.RTPWriter
+	AddStream(ssrc uint32, writer interceptor.RTPWriter)
+	SetTargetBitrate(int)
+	Close() error
+}
+
+// Stats contains internal statistics of the bandwidth estimator
+type Stats struct {
+	LossStats
+	DelayStats
+}
+
+// SendSideBWE implements a combination of loss and delay based GCC
+type SendSideBWE struct {
+	pacer Pacer
+	lc    *lossBasedBandwidthEstimator
```

Comment:
IMO code is easier to read with longer names on these. `lossController` `delayController` and `feedbackAdapter` had to scroll back a few times to find myself again.